### PR TITLE
Added Brazilian Portuguese Translations

### PR DESCRIPTION
**Pull request by LuanderFarias**

### CODE REVIEW

**Comment by lachlanjc** [comment]
#### 📝 Changed routes:


| [**/copy/pt-BR**](https://around-the-globe-9q4qilgao.hackclub.dev/copy/pt-BR) |
|:-:|
|<a href="https://around-the-globe-9q4qilgao.hackclub.dev/copy/pt-BR">Screenshot of /copy/pt-BR</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/ymxyPUSHN02D5yCJHqgjSrAbNofAK5BlPGSe_LSumaEk_M3YoXuT83Av-AZ7NEjt9Ruj6Oo0NJHraq-6q4ZnDg.png">(view full size)</a> |







Commit 877c032cf4b2130ef9e7fb7f6132af33e506683d (https://around-the-globe-9q4qilgao.hackclub.dev).

File: pages/copy/pt-BR.mdx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import { Container, Button, Heading } from 'theme-ui'
+
+<Container variant="copy">
+
+<Heading as="h1">
+  Feito por estudantes, <br />
+  para estudantes.
+</Heading>
+
+Hack Club é uma rede de clubs de ciencias da computação onde os estudantes aprendem a programar criando projetos.
```
Comment (NowDev) [inline pages/copy/pt-BR.mdx:10]:
```suggestion
Hack Club é uma rede de clubes de ciências da computação onde os estudantes aprendem a programar criando projetos.
```

**Review by NowDev — CHANGES_REQUESTED** [review]
Has some brazilian portuguese typos :)

**Comment by NowDev** [comment]
Btw, the code you've used for Brazil is actually [the ISO 639-1 code for Breton](https://www.loc.gov/standards/iso639-2/php/langcodes_name.php?iso_639_1=br),

**Comment by LucasHT22** [comment]
Eu não entendi, a página alterada seria a inicial? Se sim, tem um repositório separado, veja: https://github.com/hack-club-brasil/v1

**Review by sampoder — COMMENTED** [review]
Hey @LuanderFarias! Super appreciate the PR, however, we currently have the community maintained http://brasil.hackclub.com/ as @LucasHT22 mentioned. I'm going to close this PR to prevent duplication. Once again, thank you for the PR! Really appreciated.

**Pull request closed without merging**